### DISCUSSION TITLE
PNPM and npx create-turbo@latest

### DISCUSSION THREAD

**Discussion by indaco** [vercel/turborepo#172] [Help] [answered]
Hi,
cannot see PNPM as an option running `npx create-turbo@latest`.

```
npx create-turbo@latest

>>> TURBOREPO

>>> Welcome to Turborepo! Let's get you set up with a new codebase.

? Where would you like to create your turborepo? ./my-turborepo
? Which package manager do you want to use?
  Yarn
  Npm
```

Reading [example with PNPM](https://github.com/vercel/turborepo/tree/main/examples/with-pnpm) seems it should be on that list

```
## Setup

This repository is used in the `npx create-turbo@latest` command, and selected 
when choosing which package manager you wish to use with your monorepo (PNPM).
```

Thanks

**Comment by jaredpalmer** [comment, marked as answer]
We nixed this in a rush for launch day but need to add it back. PNPM is fully supported just need to enable it. The wrinkle is that we need to add some code to loop through all of the `package.json` `dependencies` and `devDependencies` in all of the packages and replace `"*"`  with `"workspace:*"

**Reply by ghost** [reply]
Hello,

Are you planning to provide a fix for the turborepo cli ? 
Thanks

**Reply by uwuLegacy** [reply]
so when will this be added

**Reply by jaredpalmer** [reply]
This is now available!
